File: C/1010 - Cálculo simples.c
#include <stdio.h>

int main(){

	int a, b, d, e;
	float c, f, total;

	scanf("%i %i %f", &a, &b, &c);
	scanf("%i %i %f", &d, &e, &f);

	total = b*c + e*f;

	printf("VALOR A PAGAR: R$ %.2f\n", total);

	return 0;
}

File: Go/1010 - Cálculo simples.go
package main

import (
	"fmt"
)

func main(){

	var a1, a2, b1, b2 float64
	var a3, b3, total float64

	fmt.Scan(&a1, &a2, &a3)
	fmt.Scan(&b1, &b2, &b3)

	total = a2*a3 + b2*b3

	fmt.Printf("VALOR A PAGAR: R$ %.2f\n", total)
}